src/controllers/chef.controllers.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response } from "express";
+import ChefHandler from "../handlers/chef.handlers";
+
+export const getAll = async (req: Request, res: Response) => {
+  try {
+    const chefs = await ChefHandler.getAll();
+    res.json(chefs);
+    console.log(chefs);
+  } catch (error) {
+    res.status(500).json({ message: "Error: An unexpected error occurred" });
+  }
+};
+
+export const getById = async (req: Request, res: Response) => {
+  try {
+    const chefId = req.params.id;
+    const chef = await ChefHandler.getById(chefId);
+    if (!chef) {
+      return res.status(404).json({ message: "Chef not found" });
```

Comment:
404 has not found the path (route path),  please look forward for the correct response.